### PR TITLE
[DC-1304] COPE Survey Versions for 2021Q2 CDR

### PR DESCRIPTION
**Pull request by nishanthpp93**
* [DC-1304] Update query and test

[DC-1304]: https://precisionmedicineinitiative.atlassian.net/browse/DC-1304

### CODE REVIEW

File: data_steward/cdr_cleaner/manual_cleaning_rules/survey_version_info.py [hunk 1]
```diff
@@ -36,6 +36,9 @@
       WHEN cssf.cope_month = 'may' THEN 2100000002
       WHEN cssf.cope_month = 'june' THEN 2100000003
       WHEN cssf.cope_month = 'july' THEN 2100000004
+      WHEN cssf.cope_month = 'november' THEN 2100000005
+      WHEN cssf.cope_month = 'december' THEN 2100000006
+      WHEN cssf.cope_month = 'february' THEN 2100000007
```
Comment (lrwb-aou) [inline data_steward/cdr_cleaner/manual_cleaning_rules/survey_version_info.py:41]: Cope month values should be `nov`, `dec`, and `feb`.

File: tests/integration_tests/data_steward/cdr_cleaner/manual_cleaning_rules/survey_version_info_test.py [hunk 2]
```diff
@@ -114,7 +117,13 @@ def setUp(self):
...
         'V2020.05.06' AS semantic_version,
         -- cope month provided by RDR team --
-        'may' AS cope_month)
+        'may' AS cope_month
+        UNION ALL
+        SELECT
+        700 AS participant_id,
+        30 AS questionnaire_response_id,
+        'V2020.11.06' AS semantic_version,
+        'november' AS cope_month)
```
Comment (lrwb-aou) [inline tests/integration_tests/data_steward/cdr_cleaner/manual_cleaning_rules/survey_version_info_test.py:126]: This line will need to be updated to `'nov' as cope_month)`.

**Review by lrwb-aou — CHANGES_REQUESTED** [review]
The RDR is abbreviating long month names.  We should be looking for the abbreviations.

File: data_steward/cdr_cleaner/manual_cleaning_rules/survey_version_info.py (hunk 1 above)
Comment (nishanthpp93) [reply data_steward/cdr_cleaner/manual_cleaning_rules/survey_version_info.py:41]: Thanks for catching this!

**Review by lrwb-aou — APPROVED** [review]
The integration test for this has passed.  It looks good!

**Pull request merged**